catalyst-gateway/bin/src/settings/cassandra_db.rs

```diff
@@ -69,8 +78,11 @@ impl EnvVars {
         // We can actually change the namespace, but can't change the name used for env vars.
         let namespace = StringEnvVar::new(&format!("CASSANDRA_{name}_NAMESPACE"), namespace.into());
 
-        let tls =
-            StringEnvVar::new_as_enum(&format!("CASSANDRA_{name}_TLS"), TlsChoice::Disabled, false);
+        let tls = StringEnvVar::new_as_enum(
```

Comment:
Default should be disabled.  If production needs it enabled, they should set the env var accordingly.
Otherwise we make it harder to deal with in development.

Reply:
Not resolved, `Unverified` is not the same as `Disabled` and we should have it disabled by default, and only enabled if an ENV var requires it for production?